bot/seasons/christmas/hanukkah_embed.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+import logging
+
+import requests
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+log = logging.getLogger(__name__)
+
+
+class HanukkahEmbed(commands.Cog):
+    """A cog that returns information about Hanukkah festival."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.url = ("https://www.hebcal.com/hebcal/?v=1&cfg=json&maj=on&min=on&mod=on&nx=on&"
+                    "year=now&month=x&ss=on&mf=on&c=on&geo=geoname&geonameid=3448439&m=50&s=on")
+        self.hanukkah_dates = self.get_hanukkah_dates()
+        self.hanukkah_days = []
+        self.hanukkah_months = []
+        self.hanukkah_years = []
+
+    def get_hanukkah_dates(self):
+        """Gets the dates for hanukkah festival."""
+        hanukkah_dates = []
+        r = requests.get(self.url)
+        json_data = r.json()
+        festivals = json_data['items']
+        for festival in festivals:
+            if festival['title'].startswith('Chanukah'):
+                date = festival['date']
+                hanukkah_dates.append(date)
+        return hanukkah_dates
+
+    @commands.command(name='hanukkah', aliases=['chanukah'])
+    async def hanukkah_festival(self, ctx):
+        """Tells you about the Hanukkah Festivaltime of festival, festival day, etc)."""
+        self.hanukkah_dates_split()
+
+        hanukkah_start_day = int(self.hanukkah_days[0])
+        hanukkah_start_month = int(self.hanukkah_months[0])
+        hanukkah_start_year = int(self.hanukkah_years[0])
+        hanukkah_end_day = int(self.hanukkah_days[8])
+        hanukkah_end_month = int(self.hanukkah_months[8])
+        hanukkah_end_year = int(self.hanukkah_years[8])
+
+        hanukkah_start = datetime.date(hanukkah_start_year, hanukkah_start_month, hanukkah_start_day)
+        hanukkah_end = datetime.date(hanukkah_end_year, hanukkah_end_month, hanukkah_end_day)
+        today = datetime.date.today()
+        # today = datetime.date(2019, 12, 24) (for testing)
+        day = str(today.day)
+        month = str(today.month)
+        year = str(today.year)
+        embed = Embed()
+        embed.title = 'Hanukkah Embed'
```

Comment:
I'd suggest dropping `Embed` from the title.
```suggestion
        embed.title = 'Hanukkah'
```